cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -310,6 +310,10 @@ spec:
         # Set the path where the Rook agent can find the flex volumes
         # - name: FLEXVOLUME_DIR_PATH
         #  value: "<PathToFlexVolumes>"
+        # Set the path where kernel modules can be found (defaults to /lib/modules).
+        # For example, on NixOS, the modules dir can be found here: /run/current-system/kernel-modules/lib/modules
```

Comment:
This should be kept generalized (e.g., not mentioning NixOS) as users hopefully know where this directory is for their OSes.
Would be good to move the mention that NixOS requires this to the docs.

Reply:
Thanks for the review. I've marked this WIP (which I should have done previously I suppose - sorry) while I make your suggested changes and some other changes that need to be done.